### ISSUE TITLE
[Epic] Medical records

### ISSUE THREAD

**Issue by strelichl** [department-of-veterans-affairs/va.gov-team#65243]
This epic is to track CAIA work to prepare medical records. We'll list requested updates here, and check them off as we provide guidance.

The tickets are organized by intake:
**Intake title as task list title**
- intake ticket
- associated child tickets (linked to epic, not nested under intake ticket)

Note: I've combined Phase 0 and 1

```[tasklist]
### Phase 0
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/65696
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/65697
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/65698
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/60618
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/63264
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/81736
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/83533
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/83535
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/85865
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/96184
```

```[tasklist]
### Phase 1
- [ ] Content for medical image download and viewing
- [ ] Content for date picker feature in download all records page
```

```[tasklist]
### [CAIA intake] Lab and Test Results domain
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/53759
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/57857
```

```[tasklist]
### IA design - MHV integration, entry points by launch phase
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/58406
```

```[tasklist]
### [CAIA intake] Content and IA Request for Allergies domain
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/54922
```

```[tasklist]
### [CAIA intake] Content and IA Request for Health Conditions domain
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/55310
```

```[tasklist]
### Related CC intakes
```

```[tasklist]
### CAIA Related to Medical Records
- [ ] #90780
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/65244
- [ ] #57873
- [ ] #57857
- [ ] #55310
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/55309
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/55308
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/53881
- [ ] #78985
- [ ] #78209
- [ ] #77076
- [ ] #73648
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/67990
- [ ] #58536
- [ ] #55626
- [ ] #55625
- [ ] #55621
- [ ] #55512
- [ ] #55179
- [ ] #54928
- [ ] #54131
- [ ] #53759
- [ ] #52997
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/42766
```

**Comment by coforma-jamie** [comment]
> [!NOTE]
> ## Accessibility Digital Experience (DE) Ticket Status Update
> Accessibility DE is OCTO’s new accessibility team. We’re working on a new intake process. In the meanwhile:
> - This ticket has been moved to validate and will be closed on the Accessibility DE pipeline (only).
> - This ticket will remain “open” until the work from content and IA is completed.
> 
> If you need additional accessibility support, contact the Accessibility DE team on [#accessibility-help](https://dsva.slack.com/archives/C8E985R32).
> **Thank you for working with the accessibility team!**

**Comment by mnorthuis** [comment]
closing product-specific epics
